roles/confluent.test/molecule/mtls-debian/molecule.yml

```diff
@@ -122,6 +134,8 @@ provisioner:
   inventory:
     group_vars:
       all:
+        default_internal_replication_factor: 4
```

Comment:
So the value of this PR is to make it easier to set the RF across all components? Bc you could just use the existing variables right?
```
kafka_broker_default_internal_replication_factor: 4
ksql_default_internal_replication_factor: 4
control_center_default_internal_replication_factor: 4
```

Reply:
Good point. At the beginning, I didn't know how many variables to change to align the whole services.

We can seize if it's worth adding this as a common variable.

Comparing it to other scenarios as license being set per component, we can delegate this global variable to the user and update the RF per service at the playbook level. These can be added as a use-case to the docs.

If you share this, I can rollback changes and reduce the scope to only adding the SR property, and look into the docs to get this use-case cover.

Reply:
No I'm cool with this bc the 3 should not be hardcoded everywhere anyhow :)

And UX wise, this: 
```
default_internal_replication_factor: 4
```

is easier than 
```
kafka_broker_default_internal_replication_factor: 4
kafka_connect_default_internal_replication_factor: 4
ksql_default_internal_replication_factor: 4
control_center_default_internal_replication_factor: 4
```

This is similar to making a global variable for the confluent.license property, we have that on our roadmap

Reply:
Awesome! Thanks for the tip also, will use it with the current version.